packages/ui-components/src/FuzzyTime/FuzzyTime.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FunctionComponent } from "react";
+import classnames from "classnames/bind";
+
+import { fuzzy } from "./util";
+import styles from "./FuzzyTime.module.scss";
+
+export interface FuzzyTimeProps {
+  timestamp: string | Date;
+}
+
+const cx = classnames.bind(styles);
+
+export const FuzzyTime: FunctionComponent<FuzzyTimeProps> = ({ timestamp }) => {
+  const timeago = fuzzy(new Date(timestamp));
```

Comment:
Should this component take care of parsing date from a string? Would it be better to expect the only Date and let app code to parse the string to date before passing it to the component?

Reply:
The constructor for `Date` takes [either a string or a date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date) so I thought it be more user friendly to create a date from either.

I would love to hear about your thoughts here. Do you think this degrades type safety or creates confusion for the user rather than convenience?

Reply:
Accepting String type as input will require validation if an instantiated date is valid or not. In my opinion, it's not the responsibility of FuzzyTime component to parse/validate string to date format. 
With all of this, it brings extra complexity to decide what to show if the date is invalid.

Reply:
I think that's a good point. If it somehow becomes too hard to pass a Date we can consider supporting a string, but no reason to do it until we have to.